core/extract/extract.py

```diff
@@ -48,6 +71,26 @@ def main():
         end_time,
     )
 
+ # save the extract logs to S3
+    output_directory = f'{config["workload_location"]+ "/" + extraction_name}'
```

Comment:
I feel if the workload location is not a S3 location this will fail.
we need to check if this location is local or S3 and accordingly handle these cases.

Reply:
Yes you are right. Let me just add an if clause saying if s3 then upload the logs because if the location is local we cannot do anything about it. We cannot upload logs to S3 then.

Reply:
We can add a clause in yaml to ask customers to use s3 location. At the same time we should also do error handling in the code itself. I think we can have a warning message saying no extract logs will be produced if output directory is not s3 location. Warning would be sufficient imo and we don't need to fail it. So maybe add an else statement after the 'if output_directory.startswith("s3://"):' logic?

Reply:
I agree with you. After Dev's comment I implemented the logic stated by you i`f output_directory.startswith("s3://"):`